### PR TITLE
doco (retire.md) - Extracting current epoch is outdated for network type

### PR DESCRIPTION
**Pull request by rdlrt**
For combinator networks, the slotsPerEpoch is different for Byron and Shelley eras. As such the method to simply divide current unSlot by slotsPerEpoch would only be valid for TPraos (or Byron) network. Suggest a change to query EKG instead

### CODE REVIEW

File: doc/stake-pool-operations/retire_stakepool.md [hunk 1]
```diff
@@ -7,22 +7,10 @@ To retire a pool we need to:
...
-    export CARDANO_NODE_SOCKET_PATH=relay-db/node-socket
-    cardano-cli shelley query tip --mainnet
-
-    > Tip (SlotNo {unSlotNo = 856232}) ...
-
-This gives us
-
-    expr 856232 / 21600
-    > 39
+    curl -s -H 'Accept: application/json' http://127.0.0.1:12788/ | jq -r .cardano.node.ChainDB.metrics.epoch.int.val
```
Comment (erikd) [inline doc/stake-pool-operations/retire_stakepool.md:12]: The only thing missing here is that `hasEKG` needs to be enable in the config file and that the port number needs to match.

Add a note about `hasEKG` and this is good to go.

**Review by erikd — CHANGES_REQUESTED** [review]

File: doc/stake-pool-operations/retire_stakepool.md (hunk 1 above)
Comment (rdlrt) [reply doc/stake-pool-operations/retire_stakepool.md:12]: Apologies, totally missed notification about this comment. Added now

**Review by Jimbo4350 — APPROVED** [review]

**Comment by Jimbo4350** [comment]
bors merge

**Comment by Jimbo4350** [comment]
bors merge

**Review by erikd — APPROVED** [review]

**Comment by erikd** [comment]
bors merge

**Comment by newhoggy** [comment]
bors merge

**Comment by newhoggy** [comment]
`master` already includes this text:

> You can also get the current epoch if you are monitoring a node with [EKG](../logging-monitoring/ekg.md):
> 
> ```
> curl -s -m 3 -H 'Accept: application/json' http://127.0.0.1:12788/ | jq '.cardano.node.metrics.epoch.int.val'
> > 51
> ```
> 
> So we are currently in epoch 51.

So some variant of these changes has already been merged.

**Pull request closed without merging**